treeherder/model/migrations/0020_remove_project_specific_id.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.6 on 2017-12-08 22:20
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0019_remove_job_duration'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
```

Comment:
Running `./manage.py sqlmigrate model 0020` I got:
```
BEGIN;
--
-- Alter unique_together for job (0 constraint(s))
--
CREATE INDEX `job_repository_id_20bb4d74` ON `job` (`repository_id`);
ALTER TABLE `job` DROP INDEX `job_repository_id_project_specific_id_7883a5e8_uniq`;
COMMIT;
```

...this is unfortunately buggy since the CREATE INDEX isn't needed, as the existing 5 other `index_together` entries that start with `repository_id` cover that field.

As such this removal will need a `RunSQL` that just does the ```ALTER TABLE `job` DROP INDEX `job_repository_id_project_specific_id_7883a5e8_uniq`;```.